datahub-graphql-core/src/main/resources/ingestion.graphql

```diff
@@ -196,7 +201,22 @@ type ExecutionRequest {
   Result of the execution request
   """
   result: ExecutionRequestResult
+  
+  """  
+  Ownership metadata of the IngestionSource
+  """
+  ownership: Ownership
```

Comment:
How does ExectionRequest have Ownership? I'm not seeing this field populated by your GraphQL java changes (only ingestion source)